.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+#(C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+name: ci
+
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout workspace
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.13.2
+
+      - name: Install OpenAPI Generator
+        run: npm install @openapitools/openapi-generator-cli@cli-4.2.3 -g
```

Comment:
When will be able to use a newer version of the generator?

Reply:
Will check with Dilip. The new version generates the names that conflict with existing names.

Reply:
@neilgierman in ci.yml, has listed only 'push' as the event trigger. Is that sufficient to trigger workflow even when merged with the master?

Reply:
@neilgierman Ok to mark this conversation resolved?

Reply:
This conversation started as a question about the old openapi version. If we are blocked updating to the current version ok, but this is now yet another instance where we are using an older/vulnerable version of that tool.

Reply:
I understand, but this will drag open-sourcing feature further away. I will initiate a separate thread with Dilip/Keith to sometime today.

Reply:
Ok. Just one more thing to track. Please let them know that we are continuing to propagate a bad version and we need to get our side fixed so we can use an up-to-date generator. In the mean time, you can continue with this.